pkg/export/debug/debug.go

```diff
@@ -10,18 +11,90 @@ import (
 	"github.com/grafana/beyla/pkg/internal/request"
 )
 
+// XXX deprecated (REMOVE) - use TracePrinter instead
```

Comment:
is this comment intended? what does `XXX` mean (in this context :P)

Reply:
It's a historic way to draw attention to this part of the code that needs to be fixed later (makes it easy to `grep`)

Here's an interesting read about the origins: https://www.snellman.net/blog/archive/2017-04-17-xxx-fixme/ (spoiler alert, it's definitely SFW)

Reply:
If you mean later on this PR it's fine, but if it's meant to be left here I would just use `TODO` as it's the de-factor standard in many languages (at least in Go).